server/public/plugin/hooks.go

```diff
@@ -382,4 +383,10 @@ type Hooks interface {
 	//
 	// Minimum server version: 9.5
 	OnSharedChannelsProfileImageSyncMsg(user *model.User, rc *model.RemoteCluster) error
+
+	// GenerateSupportData should be used by plugins which includes it's own content
+	// to the support packet.
```

Comment:
Other descriptions to hooks often start with `X  is invoked`. Let's use the same schema here.
```suggestion
	// GenerateSupportData is invoked when a Support Packet gets generated.
	// It allows plugins to include their own content in the Support Packet.
```